care/facility/models/patient_data.py

```diff
@@ -66,6 +70,14 @@ class PatientTeleConsultation(models.Model):
     created_by = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
 
 
+class PatientSample(FacilityBaseModel):
+    patient = models.ForeignKey(PatientRegistration, on_delete=models.PROTECT)
+    facility = models.ForeignKey("Facility", on_delete=models.PROTECT)
+    result = models.IntegerField(choices=SAMPLE_TEST_RESULT_CHOICES, default=SAMPLE_TEST_RESULT_MAP["Awaiting"])
```

Comment:
Should we map this to `PatientConsultation` too ?

Reply:
@bodhish @tomahawk-pilot  What are your thoughts? I am thinking of adding.